joomla/joomla-cms
# Issue#17932 -- Inconsistent suffix while duplicating modules

Pull Request for Issue #17932 .

### Summary of Changes
Originally: Duplicate created module (2), module (3)... and Save & Copy created module (copy)
Now: Duplicate and Save & Copy both create module (2), module (3)...
However titles created by both these buttons are independent, i.e., duplicating module will give module (1) and Save & Copy of module will also give module (1) (irrespective of whether it exists or not).

### Testing Instructions
In Module Manager select/check a module.
-     Click button Duplicate.
-     The title of copy has a postfix (2).
-     Now open a module by clicking the title link.
-     Click button Save as Copy.
-     The title of copy gets a postfix (copy).


### Expected result
Uniform module title.


### Actual result
As mentioned above


### Documentation Changes Required
None.
